platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/AbstractTransformExecution.java

```diff
@@ -142,6 +146,9 @@ private WorkOutput executeWithinTransformerListener(ExecutionRequest executionRe
             @Override
             public TransformExecutionResult call(BuildOperationContext context) {
                 try {
+                    if (LOGGER.isInfoEnabled()) {
```

Comment:
💭 Should we reduce that to debug?

Reply:
Yeah, that seems reasonable. Do we have some integration tests depending on this? Probably not (and nor should they).

Reply:
We had some, but I fixed that too